index.html

```diff
@@ -0,0 +1,61 @@
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+
+
+
+<main id="main">
+
+<h1 id="title">Dr. Norman Borlaug</h1>
+    
+    <div id ="img-div">
+
+        <img  id="image" src="dr_image.jpg" alt=""> 
+
+        <figcaption id="img-caption"> Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</figcaption>
+    </div>
+    <div id="tribute-info">
+<h2> Here's a time line of Dr. Borlaug's life:
+</h2>
```

Comment:
Kindly use correct HTML5 semantic tags (e.g., header, section, footer, nav) over divs.

Reply:
[optional] **Recommended** Please try to be explicit in giving reasons why you want the student to make a change as it would better help the student to better understand the required change and fix it.